security/subsystem/src/main/java/org/jboss/as/security/MappingProviderModuleDefinition.java

```diff
@@ -40,4 +55,29 @@
     public AttributeDefinition[] getAttributes() {
         return ATTRIBUTES;
     }
+
+    static void registerTransformers_1_3_0(ResourceTransformationDescriptionBuilder parentBuilder) {
+        final String defaultModuleName = ModuleName.PICKETBOX.getName();
+        ResourceTransformationDescriptionBuilder builder = parentBuilder.addChildResource(PATH_PROVIDER_MODULE);
+        builder.getAttributeBuilder()
+                .setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(false, false,
+                        new ModelNode(/* undefined */), new ModelNode(defaultModuleName)), MODULE)
+                .addRejectCheck(new RejectAttributeChecker.DefaultRejectAttributeChecker() {
```

Comment:
> it should be enough to just use RejectAttributeChecker.DEFINED? Unless I am missing something, or you really really want the extra log message.

Yes, I like your proposal. It is simpler and the only diff is the log message. Let me try it.